tests/base/class-aioseop-notices-testcase.php

```diff
@@ -490,31 +490,31 @@ protected function data_screens_wp() {
 				'url'       => site_url() . '/wp-admin/post-new.php',
 				'dir'       => WP_DEVELOP_DIR . '/src/wp-admin/post-new.php',
 			),
-//			array(
-//				'screen_id' => 'post',
-//				'url'       => site_url() . '/wp-admin/post.php?post=###&action=edit',
-//				'dir'       => WP_DEVELOP_DIR . '/src/wp-admin/post.php?post=###&action=edit',
-//			),
+			// array(
```

Comment:
isnt it better to remove this than comment it out?

Reply:
Not quite yet. I still have other related issues/PRs that I got to add soon, and is still very useful information. It would also save a lot of time from having to investigate all the screens needed.